index.html

```diff
@@ -53,9 +53,9 @@ <h3>
               <button class="btn">SELECT</button>
           </div>
           <div class="price_style">
-            <h3>
+            <div>

```

Comment:
- [ ] You should use a `h1` tag for your titles, using a '<div>' for titles not semantic HTML

Reply:
Please refer to the comment above for the same issue 👆.